install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -2560,6 +2560,9 @@ envoy:
 # -- Enable/Disable use of node label based identity
 nodeSelectorLabels: false
 
+# To include or exclude matched resources from cilium node identity evaluation
```

Comment:
Can you expand on this a bit? Is this a bool? Or a list of labels?

Reply:
list of labels, just like the `--labels` flag

Reply:
Could you mention that in the comment? 🙏

Reply:
sure